data_converters/lum_kitti_object.py

```diff
@@ -0,0 +1,81 @@
+'''
+    Given an object entry construct a kitti object for easy retrieval of object properties
+'''    
+import numpy as np
+
+class lum_kitti_object():
```

Comment:
```suggestion
class KittiObject():
```
Suggestion: Fix case. Also lum_prefix is usually a part of package names, Not needed in class/function/variable names